spec/duration.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<meta charset="utf8" />
+
+<emu-clause id="sec-temporal-duration">
+    <h1>Duration Objects</h1>
+    <p>
+    Duration objects describe the difference between two like Date, Time or DateTime objects.
+    They are only created via the <i>.minus()</i> methods of these objects.
+    </p>
+    <ul>
+        <li>The <i>.minus()</i> operation on Date objects, creates a Duration that only has years, months and days properties.</li>
+        <li>The <i>.minus()</i> operation on Time objects, creates a Duration that only has hours, minutes, seconds, milliseconds, microseconds and nanoseconds properties.</li>
+        <li>The <i>.minus()</i> operation on Instant objects, creates a Duration like that of Time objects.</li>
+        <li>The <i>.minus()</i> operation on DateTime objects , creates a Duration that has all member properties, except if passed an option indicating it should be created as a Time object or Date object.</li>
+    </ul>
+    <p>
+    When created, the smaller units are combined into full larger units (i.e. 1000 milliseconds are recorded as 1 seconds) up to the largest available unit.
+    The result of this is that the largest available unit may have a larger value in some circumstances. Durations created from Time objects may have an hours property larger than 24, which would be invalid for a Duration created from a DateTime object.
+    </p>
+    <p>
+    When a Duration is applied to an object via the <i>.plus()</i> method, only the relevant fields are taken into consideration.
+    Such that a Date objects receiving a DateTime duration will entirely ignore the values of the hours and below fields. As a consequence
+    A Date object receiving a Duration from the <i>.minus()</i> of Time objects, will be a no-op, since it will not have any relevant fields even in the case where the hours are larger than 24 which would equate
```

Comment:
```suggestion
    A Date object receiving a Duration from the <i>.minus()</i> of Time objects will be a no-op, since it will not have any relevant fields even in the case where the hours are larger than 24 which would equate.
```